### PR TITLE
replacement for make

### PR DESCRIPTION
**Pull request by orcema**
make didn't work on windows 10, thus i setup tasks for reproducing the make instructions.
shortcut in VSCode for displaying all tasks:  `ctrl+alt+t`

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "problemMatcher": []
+      },
+      {
+        "label": "gunicorn, hot-reload and VS Code debugger",
+        "type": "shell",
+        "command": "docker-compose run --rm -e DEBUGGER=True --service-ports flask-server gunicorn --reload --bind 0.0.0.0:5000 --timeout 3600 app:app",
+        "problemMatcher": []
+      }
+    ]
+  }
```
Comment (adriencaccia) [inline .vscode/tasks.json]: Could you add an empty line at the end of file ?

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (adriencaccia) [inline .vscode/tasks.json:1]: Could you rename the commit with something more explicit, like `Add VS Code tasks` ?

**Review by adriencaccia — CHANGES_REQUESTED** [review]
Thank you very much for this PR :heart: !

Can you also add a `For Windows users` section in the `README.md`, explaining how to execute the tasks ?

**Comment by orcema** [comment]
i will setup a new pull request

**Pull request closed without merging**